Editor/Modules/TextureModule.cs

```diff
@@ -42,11 +44,24 @@ class TextureModule : ProjectAuditorModule
         };
         public override string name => "Textures";
 
+        List<ITextureAnalyzer> m_Analyzers;
+        HashSet<ProblemDescriptor> m_ProblemDescriptors;
+
         public override IReadOnlyCollection<IssueLayout> supportedLayouts => new IssueLayout[]
         {
             k_TexturesIssueLayout,
         };
 
+        public override IReadOnlyCollection<ProblemDescriptor> supportedDescriptors => m_ProblemDescriptors;
+
+        public override void Initialize(ProjectAuditorConfig config)
+        {
+            m_Analyzers = new List<ITextureAnalyzer>();
+            m_ProblemDescriptors = new HashSet<ProblemDescriptor>();
+
+            foreach (var type in TypeCache.GetTypesDerivedFrom(typeof(ITextureAnalyzer)))
+                AddAnalyzer(Activator.CreateInstance(type) as ITextureAnalyzer);
+        }
 
         public override void Audit(ProjectAuditorParams projectAuditorParams, IProgress progress = null)
         {
```

Comment:
let's invoke Analyze for all registered ITextureAnalyzer's on all textures

Reply:
Analyze now run for each registered iTextureAnalyzer on each texture